Documentation/security/network/encryption-wireguard.rst

```diff
@@ -283,6 +283,13 @@ options:
   key in its corresponding ``CiliumNode`` CRD when a worker node's public key
   changes, given that the worker node will be unable to do so itself.
 
+  N/S Loadbalancer traffic won't be encrypted when an intermediate node redirects
+  a request to another node where a selected backend is running,
+  with the following configuration:
+
+  - LoadBalancer & NodePort XDP Acceleration
+  - Direct Server Return (DSR)
```

Comment:
Edits for clarity, use present tense, use clear, plain language
```suggestion
  N/S load balancer traffic isn't encrypted when an intermediate node redirects
  a request to a different node with the following backend configuration:

  - LoadBalancer & NodePort XDP Acceleration
  - Direct Server Return (DSR)
```

Reply:
Thank you for your comment! I have updated the doc accordingly.